setup.py

```diff
@@ -79,6 +79,7 @@
         'bitcoin==1.1.42',
         'flask==0.10.1',
         'requests==2.9',
+        'gunicorn',
```

Comment:
@sbellem wrote:
> it appears that inequality operators (<, <=, >, >=) are preferred

check out my last comment, the `~` operator is gooood! :dancers: